python-test/features/steps/control_plane_agents.py

```diff
@@ -231,7 +239,7 @@ def reset_agent_remotely(context):
 
 
 @threading_wait_until
-def expect_container_status(token, agent_id, status, event=None):
+def expect_agent_status(token, agent_id, status, event=None):
```

Comment:
I would add `wait` to the name of the function to distinguish in the name that is a longer duration func.

Reply:
check again please